recipes/xorg-libximerna/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.1.4" %}
+
+package:
+  name: xorg-libximerna
+  version: {{ version }}
+
+source:
+  url: https://gitlab.freedesktop.org/xorg/lib/libxinerama/-/archive/libXinerama-{{ version }}/libxinerama-libXinerama-{{ version }}.tar.gz
+  sha256: d9fb7f224281f5467d36f54b01a8f49e866a994410d5bce3b019f2da8bf8e643
+
+build:
+  number: 1
+  skip: True   # [not linux]
+  run_exports:
+    - {{ pin_subpackage('xorg-libximerna', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+  host:
+    - xorg-util-macros >1.8
+    - xorg-libx11 >=1.6
+    - xorg-xineramaproto >=1.1.99.1
+    - xorg-libxext
+  run:
+    - xorg-libxext
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libXinerama.so
+    - conda inspect linkages -p $PREFIX $PKG_NAME
+
+about:
+  home: https://gitlab.freedesktop.org/xorg/lib/libxinerama
+  license: MIT
+  license_family: MIT
+  license_file: COPYING
+  summary: 'API for Xinerama extension to X11 Protocol.'
```

Comment:
I see that this summary comes from the library README, but I don't think it's great. I'd put something more like "Client library for the Xinerama extension to the X11 protocol."

Reply:
Thanks. I really don't know what this library does, but it helped me compile my other library.